app/Platform/Events/AchievementCreated.php

```diff
@@ -17,7 +17,7 @@ class AchievementCreated
     use SerializesModels;
 
     public function __construct(
-        public Achievement $achievement
+        public Achievement|int $achievement
```

Comment:
Absolutely, those types will all be reduced to just model classes.
I had to allow for more during development because in some areas where those events are dispatched the respective models were not present yet.
Will be cleaned up in one of the upcoming commits.

Reply:
Cleaned up in https://github.com/RetroAchievements/RAWeb/commit/6263982769e2405a1f263d4a187e4c3498542809